pkg/services/ngalert/metrics/ngalert.go

```diff
@@ -45,13 +45,15 @@ type NGAlert struct {
 }
 
 type Scheduler struct {
-	Registerer               prometheus.Registerer
-	BehindSeconds            prometheus.Gauge
-	EvalTotal                *prometheus.CounterVec
-	EvalFailures             *prometheus.CounterVec
-	EvalDuration             *prometheus.SummaryVec
-	GetAlertRulesDuration    prometheus.Histogram
-	SchedulePeriodicDuration prometheus.Histogram
+	Registerer                prometheus.Registerer
+	BehindSeconds             prometheus.Gauge
+	EvalTotal                 *prometheus.CounterVec
+	EvalFailures              *prometheus.CounterVec
+	EvalDuration              *prometheus.SummaryVec
+	SchedulePeriodicDuration  prometheus.Histogram
+	SchedulableAlertRules     prometheus.Gauge
+	SchedulableAlertRulesHash prometheus.Gauge
```

Comment:
nit: this is probably fine, but I can see that you chose to use "schedulable" as a prefix here but not for `UpdateAlertRulesDuration` which seems inconsistent?